Exdi/exdigdbsrv/ExdiGdbSrv/ReadMe.txt

```diff
@@ -0,0 +1,83 @@
+========================================================================

```

Comment:
There are a bunch of instances here not changed from "sample". I'd say just delete the file.

Reply:
Yes, the readme.txt can be deleted